pkg/strategy/grid2/strategy.go

```diff
@@ -1800,6 +1804,57 @@ func (s *Strategy) getWriteContext(fallbackCtxList ...context.Context) context.C
 	return context.Background()
 }
 
+type orderPollerRequirement interface {
+	types.ExchangeOrderQueryService
+	types.ExchangeTradeHistoryService
+}
+
+type OrderPoller struct {
+	filledOrders *bbgo.OrderStore
+	lastSyncTime time.Time
+	service      orderPollerRequirement
+}
+
+func newOrderPoller(symbol string, service orderPollerRequirement) *OrderPoller {
+	orderStore := bbgo.NewOrderStore(symbol)
+	orderStore.AddOrderUpdate = true
+	orderStore.RemoveFilled = false
+	return &OrderPoller{
+		filledOrders: orderStore,
+	}
+}
+
+func (p *OrderPoller) Bind(stream types.Stream) {
+	p.filledOrders.BindStream(stream)
+}
+
+func (p *OrderPoller) Run(ctx context.Context, interval time.Duration) {
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-ticker.C:
+
```

Comment:
lack of operation?

Reply:
oh wait that should be in another branch

Reply:
fixed